### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by elboyran**
Added How to use DIANNA section and a high-level architecture. Solves #193

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -71,6 +71,15 @@ To install the most recent development version directly from the GitHub reposito
 python3 -m pip install git+https://github.com/dianna-ai/dianna.git
 ```
 
+## How to use DIANNA
+To use DIANNA you need a _trained AI model_ (in ONNX format) and a _data item_ (e.g. an image or text, etc.) for which you would like to explain the output of the model. 
+DIANNA calls an explaianble AI method to produce a "heatmap" of the relevances of each data pont (e.g. pixel, word) to a given model's decision overlaid on the data item.
```
Comment (cwmeijer) [inline README.md:76]:
```suggestion
DIANNA calls an explainable AI method to produce a "heatmap" of the relevances of each data pont (e.g. pixel, word) to a given model's decision overlaid on the data item. 
```

Comment (cwmeijer) [inline README.md:76]:
```suggestion
DIANNA calls an explaianble AI method to produce a "heatmap" of the relevances of each data point (e.g. pixel, word) to a given model's decision overlaid on the data item. 
```

**Review by cwmeijer — APPROVED** [review]
Great addition! You may want to include an extra line or link to our onnx conversion scripts in the text because you mention it also in the figure you added. Either way, it's very much an improvement.

**Comment by elboyran** [comment]
Something wring with the keras to onnx tutorial though, to be fixed.

**Comment by egpbos** [comment]
Could you still fix the spell error suggested above, pont vs point?

I made #201 as my review on this PR, up for discussion, of course :)

**Pull request merged**